install/InstallHalyard.sh.gen

```diff
@@ -388,14 +389,23 @@ install_halyard
 
 configure_bash_completion
 
-start halyard
-
 printf 'Waiting for the Halyard daemon to start running'
 
+WAIT_START=$(date +%s)
+
 set +e 
 hal --ready &> /dev/null
 
 while [ "$?" != "0" ]; do
+  WAIT_NOW=$(date +%s)
+  WAIT_TIME=$(( $WAIT_NOW - $WAIT_START ))
+
+  if [ "$WAIT_TIME" > "$HALYARD_STARTUP_TIMEOUT_SECONDS" ]; then
```

Comment:
No you're right it should be `-gt`

I figured counting approximations would break if someone changed the sleep time per iteration. This felt a little more consistent.